build/configs/artik053/audio/defconfig

```diff
@@ -1207,9 +1244,9 @@ CONFIG_EXAMPLES_SLSIWIFI_STACKSIZE=2048
 # Shell
 #
 CONFIG_TASH=y
-CONFIG_TASH_MAX_COMMANDS=128
+CONFIG_TASH_MAX_COMMANDS=132
```

Comment:
@btheosam @Taejun-Kwon Please give  your views

Reply:
All these are based on artik053/tc, and I added just audio feature.
For now, audio testing need fs & many of kernel features.
Futhermore, somebody will test audio with network sending as we know, that's why I decided keep as same as tc because if it enable we can use most of feature.
As possible as we can, we will keep current config but if there's any config which is not really neccessary let us know that.

Reply:
TC has many unnecessary configs to run unit testcase. Based on TC is not good I think. Let's use nettest or wifi_test.